pygeoapi/api.py

```diff
@@ -1155,7 +1161,7 @@ def describe_collections(self, request: Union[APIRequest, Any],
                     'href': f'{self.get_collections_url()}/{k}/schema?f={F_HTML}'  # noqa
                 })
 
-            if collection_data_type in ['feature', 'record', 'tile']:
+            if is_vector_tile or collection_data_type in ['feature', 'record']:
```

Comment:
if a vector tile is configured like this:

```
   providers:
       - type: tile
         name: MVT-proxy
         data: http://localhost:7800/public.ne_50m_admin_0_countries/{z}/{x}/{y}.mvt
            options:
              zoom:
                min: 0
                max: 15
             schemes:
                 - WebMercatorQuad
         format:
             name: pbf
             mimetype: application/vnd.mapbox-vector-tile
```

it's type should be `tile`.

https://github.com/geopython/pygeoapi/blob/3ce68acdb85ec655e088b96f92b436f021beb57b/pygeoapi/api.py#L1039

I needed to distinguish between raster tiles and vector tiles because many metadata links do not apply to raster tiles. If there is a better way, please tell.

Reply:
(I removed `tile` from the condition)

Reply:
@simonseyock that is a fair observation, but  `type` and `data_type` are two different things. `type`, as defined in the configuration file refers to the standard (in this case, 'OGC API - Tiles'), while `data_type` refers to the actual data.  OGC API - TIle supports several types of data: maps, features, coverages.
Regarding your code, removing the `is_vector_tile` from the condition, would avoid these links from being generated in the case of raster tiles, while still allowing it to run with vector tiles.

Reply:
but `collection_data_type` is not the `data_type` it is the `type` of the `collection_data` - which seems to be the provider.

https://github.com/geopython/pygeoapi/blob/3ce68acdb85ec655e088b96f92b436f021beb57b/pygeoapi/api.py#L1038-L1039

Reply:
or do I misread that? I do not have a vector tile source to test this, if you can provide a public one with the matching configuration in pygeoapi I can test it.

Reply:
@simonseyock you can debug it with these examples:
https://github.com/geopython/pygeoapi-examples/tree/main/docker/mvt-tippecanoe
https://github.com/geopython/pygeoapi-examples/tree/main/docker/mvt-elastic

Reply:
@doublebyte1

`collection_data_type` is `tile` for vector tiles, as this is only the provider config. This code was faulty still, I opened this new PR: https://github.com/geopython/pygeoapi/pull/1575